spec/testfiles/syntax_test_TheBigTestFile.ps1

```diff
@@ -1110,6 +1110,12 @@ $b -cLike $c
 #                   ^ keyword.operator.string-format.powershell
 
 # Misc test cases
+Test-Function -Class ClassName
+#              ^ not:storage.type.powershell
+New-Object -TypeName System.Diagnostics.Process
+#                                       ^ not:keyword.control.powershell
+New-Object -TypeName System.Data
+#                           ^ not:keyword.control.powershell
```

Comment:
❤️

Reply:
@vors BTW I'm almost done with PR for #104